static/docs/commands-reference/diff.md

```diff
@@ -0,0 +1,44 @@
+# diff
+
+Show changes between files or directories that are under DVC control.
+
+## Synopsis
+
+```usage
+  usage: dvc diff [-h] [-q | -v] [-t TARGET] a_ref [b_ref]
+
+  positional arguments:
+    a_ref                 Git reference from which diff calculates
+    b_ref                 Git reference until which diff calculates,
+                          if omitted diff shows the difference
+                          between current HEAD and a_ref
+```
+
+## Description
+
+Given two Git commit references (commit hash, branch or tag name, etc) `a_ref`
+and `b_ref`, this command shows a standard textual comparisson (or "diff") of
+all the files under DVC control among both refs. This is similar to how `git
```

Comment:
it does not actually, I think all it can do right now is to compare number of files, total size cc @efiop

Reply:
Ah sorry I haven't actually tested this first version of the description ha sry. Will update for next PR

Reply:
OK, fixed this now actually.